rotkehlchen/exchanges/binance.py

```diff
@@ -589,6 +607,140 @@ def _query_lending_balances(
 
         return balances
 
+    def _query_lending_interests_history(
+            self,
+            cursor: 'DBCursor',
+            start_ts: Timestamp,
+            end_ts: Timestamp,
+    ) -> None:
+        """Queries Binance lending interest history, transforms it
+        into `HistoryBaseEntry` objects and save it in the database.
+        May raise:
+        - RemoteError
+        """
+        ranges = DBQueryRanges(self.db)
+        range_query_name = f'{self.location}_history_events_{self.name}'
+        ranges_to_query = ranges.get_location_query_ranges(
+            cursor=cursor,
+            location_string=range_query_name,
+            start_ts=start_ts,
+            end_ts=end_ts,
+        )
+        for query_start_ts, query_end_ts in ranges_to_query:
+            events = []
+            for lending_type in ('DAILY', 'ACTIVITY', 'CUSTOMIZED_FIXED'):
+                response = self._api_query_list_within_time_delta(
+                    api_type='sapi',
+                    start_ts=query_start_ts,
+                    time_delta=BINANCE_LENDING_INTEREST_HISTORY_INTERVAL_TS,
+                    end_ts=query_end_ts,
+                    method='lending/union/interestHistory',
+                    additional_options={'lendingType': lending_type, 'size': 100},
```

Comment:
how was this size chosen?

Reply:
I basically chose the max size i.e. 100

Plus since this is used for pagination as a result of `current` query parameter, it'll reduce number of calls

Reply:
Can you state that the timedelta is because

```
The time between startTime and endTime cannot be longer than 30 days.
```

https://binance-docs.github.io/apidocs/spot/en/#get-interest-history-user_data-2

Is the kind of thing that you look later and forget why is like this.

And okey, the 100 is because is stated in the docs that is the maxium